lib/provider/aws/lambda-edge-origin-request/clean-up-event.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+function getPath({ uri }) {
+  return typeof uri === 'string' ? uri : '/';
+}
+
+module.exports = function cleanupEvent(evt, options) {
+  const event = (evt &&
+    evt.Records &&
+    evt.Records.length > 0 &&
+    evt.Records[0].cf) ||
+    {};
+
+  event.config = event.config || {};
+  
+  event.request = event.request || {};
+  event.request.body = event.request.body || '';
```

Comment:
The request body in a Lambda@Edge event is an object, not a string.

[See here:](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c72d502e2e339e17f374842977c78a8b18bc4e0c/types/aws-lambda/common/cloudfront.d.ts#L45-L50)

```typescript

export interface CloudFrontRequest {
    body?: {
        action: 'read-only' | 'replace';
        data: string;
        encoding: 'base64' | 'text';
        readonly inputTruncated: boolean;
    };
    readonly clientIp: string;
    readonly method: string;
    uri: string;
    querystring: string;
    headers: CloudFrontHeaders;
    origin?: CloudFrontOrigin;
}
```